### ISSUE TITLE
feature request: TOP values

### ISSUE THREAD

**Issue by tomasz13** [fhorinek/SkyDrop#73]
It would be fine to see top reached values immediately after flight. (max. altitude; max. climb; max. sink; max & min speed; etc.)

**Comment by colignym** [comment]
Add min altitude to that,  nice to see how low was the low save....

**Comment by SergeantAsh** [comment]
And most importantly, SkyDrop will SAVE these values and store them on the SD card after each flight,

**Comment by leachim1** [comment]
+1

**Comment by fhorinek** [comment]
so far it will display 
min/max alt, climb and ftime
no save of speed for now

**Comment by SergeantAsh** [comment]
Nice work! Does it write these stats to the SD too for reference later? I'd love to see it create a folder for each date/flight with all of the relevant details inside it?

**Comment by tomasz13** [comment]
saving stats to the SD would be nice :)

**Comment by Flyingplanet** [comment]
I don't understand how this is closed? Right now, these values are displayed just after the flight, but I don't know how to display them again without a PC (maybe I missed something, how can I recall these values for the flight directly on the skydrop? How can I read the values from the 2 last flight I did 3 days ago). This feature is a basic feature on any vario (my first vario had it, and I bought it in 2002...) and right now, it's only half finished on the Skydrop.

Also #180 is a bit different, as I suggested to offer a logbook on the skydrop itself like Reversale does, to provide analytics of your flights
